packages/core/src/initialization/network-options.ts

```diff
@@ -22,6 +23,7 @@ const ALLOW_CUSTOM_TOPIC = [Networks.INMEMORY, Networks.LOCAL]
 export type CeramicNetworkOptions = {
   name: Networks // Must be one of the supported network names
   pubsubTopic: string // The topic that will be used for broadcasting protocol messages
+  offset: number // A random offset that is added to the pubsub topic to isolate local/inmemory deployments
```

Comment:
The event id needs a network id if using a local network. 

Technically for rust-ceramic nodes we run in tests, the networkId is 0. 

For rust-ceramic nodes that are run remotely, we will need a way to retrieve the network info from the recon node, or add configuration flags/options to set the network id. I'd prefer to retrieve the network info from the recon node to ensure no misconfiguration.

Since we do not have a way to retrieve the network info from the recon node, i'll remove this and add a way to configure it similar to the pubsub network topic

Reply:
Note I am not giving anyway for the cli to set the networkId. The assumption is it will be 0, and we can decide how we want to set it later.